### DISCUSSION TITLE
Switch for compass sensor necessary?

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#673] [Q&A] [answered]
As startet in the discussion #667 there is the question whether we really need the switch for the compass sensor in the map settings.
If switching off saves energy, then it is important (and the energy saving should be mentioned in the description).
If not, there is no need for this switch, imho🙂. You can use compass functionality in the map or you don't use it if you are not  interested. But you  don't need the switch.
What do you think?

**Comment by devemux86** [comment]
@kds112 Thanks for the suggestion.

I have not yet examined all the details.
If it doesn't affect the battery, we can remove the switch.

**Comment by devemux86** [comment]
Compass sensor setting was removed in **[Cruiser 5.1.0-b3](https://github.com/devemux86/cruiser/discussions/671)**.

**Comment by devemux86** [comment, marked as answer]
- Compass sensor setting was removed in https://github.com/devemux86/cruiser/discussions/684
